01week/rockPaperScissors.js

```diff
@@ -7,17 +7,56 @@ const rl = readline.createInterface({
   output: process.stdout
 });
 
+//Both Player 1 and Player 2 will each pick a valid object(string) to play
+  //If both players enter valid object then show game result
+  //If one or both players enters an invalid object to play inform user "Please choose valid play"
+  //If player1 picks rock and player2 picks paper then display "Player 2 wins"
+  //If player1 picks rock and player2 picks scissors then display "Player 1 wins"
+  //If player1 picks paper and player2 picks rock then display "Player 2 wins"
+  //If player1 picks paper and player2 picks scissors then display "Player 2 wins"
+  //If player1 picks scissors and player2 picks rock then display "Player 2 wins"
+  //If player1 picks scissors and player2 picks paper then display "Player 1 wins"
 
-function rockPaperScissors(hand1, hand2) {
 
-  // Write code here
+  function rockPaperScissors(handOne, handTwo) {
+// Write code here 
+  //If both players choose the same object to play then display "It is a draw"
+      let hand1 = handOne.toLowerCase();
+      let hand2 = handTwo.toLowerCase(); 
+      if (hand1 !== "rock" && hand1 !== "paper" && hand1 !=="scissors" || hand2 !== "rock" && hand2 !== "paper" && hand2 !== "scissors"){
 
+        console.log("Please choose rock, paper or scissors");
```

Comment:
Return he string, the getPrompt function console logs the return. No need to call getPrompt()